packages/shopify-codemod/transforms/remove-return-voids.js

```diff
@@ -0,0 +1,34 @@
+import {matchLast} from './utils';
+
+export default function removeReturnVoids({source}, {jscodeshift: j}, {printOptions = {}}) {
```

Comment:
Name might make more sense as `removeTrailingElseUndefinedReturn` or something similar, given the scope of what it's solving (if you do change the name, make sure to change all the file names too).